src/EditorFeatures/DiagnosticsTestUtilities/CodeActions/AbstractCodeActionOrUserDiagnosticTest_TestAddDocument.cs

```diff
@@ -28,10 +28,15 @@ public abstract partial class AbstractCodeActionOrUserDiagnosticTest
                 initialMarkup, expectedMarkup,
                 expectedContainers, expectedDocumentName,
                 WithRegularOptions(parameters));
-            await TestAddDocument(
-                initialMarkup, expectedMarkup,
-                expectedContainers, expectedDocumentName,
-                WithScriptOptions(parameters));
+
+            // VB script is not supported:
```

Comment:
❔ How did this not fail before?

Reply:
Lucky? VS script kind of works, but not really.

Reply:
I think most of these will go away as we switch to the new testing library.